qa/os/src/test/java/org/elasticsearch/packaging/test/PackageTests.java

```diff
@@ -49,9 +50,6 @@
 import static org.elasticsearch.packaging.util.Packages.assertRemoved;
 import static org.elasticsearch.packaging.util.Packages.installPackage;
 import static org.elasticsearch.packaging.util.Packages.remove;
-import static org.elasticsearch.packaging.util.Packages.restartElasticsearch;
-import static org.elasticsearch.packaging.util.Packages.startElasticsearch;
-import static org.elasticsearch.packaging.util.Packages.stopElasticsearch;
```

Comment:
Why were all these removed? Seems to have created more diffs in the file than necessary.

Reply:
There is now a name conflict with `startElasticsearch` and `stopElasticsearch` in the superclass. I can clean up the diff a little bit by calling the instance methods, which will immediately delegate to these removed static methods.

Reply:
We have agreed in slack to save renaming methods for a potential follow-up PR.